### ISSUE TITLE
[Research] WCAG 2.1 Rule: Create rules for Pointer Gestures - 196

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#589]
Determine which rules we can create for detection of pointer gesture issues. See Box for background:
https://ibm.ent.box.com/file/521542224211

——————
- [x] research previous notes, possible ACT rules
- [na] general rule description (SME & Dev)
- [na] testcases written (Dev)
- [na] rule developed (Dev)
- [na] message written (SME)
- [na] help written (SME)

Original internal issue:  https://github.ibm.com/IBMa/accessibility-web-engine/issues/196

**Comment by marcjohlic** [comment]
From @tombrunet 5/21
In order to do this, I think we need to have good detection of relevant registered events. In the past, we've found that analytics tools register events for the sake of watching and not for the purpose of performing actions, which caused us to over report errors. I don't believe we have a good way to implement these rules at this time.

**Comment by marcjohlic** [comment]
From @shunguoy 8/21
 Rule discussion conclusion:  https://ibm.ent.box.com/notes/839472075014
Not high priority due to the fact that there is no reliable way to detect pointer gesture events, or to detect if there are alternatives to a complex gestures.
What to do:  
1. no need to update the toolkit that already has the requirements for dev/design/test: https://www.ibm.com/able/requirements/requirements/#2_5_1
2. move the issue to the backlog until we have a better approach in the future.

**Comment by marcjohlic** [comment]
one of the goals or outcomes of this initial analysis and squad meeting is to determine what should be the priorities of any possible rule development actions now that we are getting close to the end of the release.

**Comment by philljenkins** [comment]
Charu had aslo done some initial work documenting the need for this new rule, in a word doc
[Rule-pointer-gestures.docx](https://github.com/IBMa/equal-access/files/7824499/Rule-pointer-gestures.docx)

**Comment by philljenkins** [comment]
ACT rules (as of 1-6-2022):

- No ACT rules mention support for 2.5.1 Pointer Gestures 
- No ACT rules mention techniques:

[G215: Providing controls to achieve the same result as path based or multipoint gestures](https://www.w3.org/WAI/WCAG21/Techniques/general/G215)
[G216: Providing single point activation for a control slider](https://www.w3.org/WAI/WCAG21/Techniques/general/G216)
[F105: Failure of Success Criterion 2.5.1 due to providing functionality via a path-based gesture without simple pointer alternative](https://www.w3.org/WAI/WCAG21/Techniques/failures/F105)

**Comment by philljenkins** [comment]
Analysis review with squad help Fri 1-7-2021

**Comment by philljenkins** [comment]
Technical assessment is that because the requirement applies to web content (web apps) that interpret pointer actions, for example unique web apps supporting any or all of the following: 
- mouse pointer, 
- touch on touch screen or trackpad, 
- stylus input, 
- laser pointer input

and because this does **_not_** apply to actions that are required to operate the browser/user agent or assistive technology:

* Detecting gesture events specific in web content/apps is not possible (W3C has not established a standard yet)
* No ACT rules exist

While mouse pointer related checking/rules exist, flagging **_all_** mouse events as _potential gestures for multi-path_ "Needs Review" would negatively create an inordinate amount of time on task for all teams, resulting in dissatisfaction from the Checker users.
